devices/grlib-multi/test-grlib-multi.c

```diff
@@ -775,6 +775,5 @@ void runner(void)
 
 int main(int argc, char *argv[])
 {
-	UnityMain(argc, (const char **)argv, runner);
-	return 0;
+	return UnityMain(argc, (const char **)argv, runner) == 0 ? EXIT_SUCCESS : EXIT_FAILURE;
```

Comment:
I know that this version is also correct, but to make it more clear I would add parentheses:
```suggestion
	return (UnityMain(argc, (const char **)argv, runner) == 0) ? EXIT_SUCCESS : EXIT_FAILURE;
```